scripts/aot.spec.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Let's not call this a spec file, some globs might match it based on `*.spec` to include in karma or something.

Reply:
I will rename.